changelog/2793.feature.rst

```diff
@@ -0,0 +1 @@
+Now able to create a Map using an array and a WCS object.
```

Comment:
This should read 
```
`astropy.wcs.WCS`
```

Reply:
Sorry I didn't get that.
If I change the name of the file the snupy bot gives an error

Reply:
I think he was suggesting that `WCS` should be
```
`astropy.wcs.WCS`
```
The same for `Map` , 
```
`sunpy.map.Map`
```

Reply:
Oh ok ok I will do it

Reply:
Done